### DISCUSSION TITLE
Do we use use bundle-build image ?

### DISCUSSION THREAD

**Discussion by cscetbon** [cscetbon/cassandra-k8s-operator#62] [Q&A] [answered]
@AKamyshnikova what is https://github.com/cscetbon/casskop/blob/master/shared.mk#L48-L50 used for ? I think it's not used anywhere. Can we just get rid of it ?

**Comment by AKamyshnikova** [comment, marked as answer]
yep, this can be dropped. It was autogenerated.

**Comment by cscetbon** [comment]
@AKamyshnikova Can you also tell me if https://github.com/cscetbon/casskop/blob/master/bundle/manifests/casskop.clusterserviceversion.yaml#L40 is needed in this file. I'm trying to get rid of the version in all templates and set it when it gets packaged

**Reply by cscetbon** [reply]
Never mind I got my answer. Everything can be regenerated so getting rid of bundle folder as well
